front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -14,45 +14,53 @@ import TimesStops from './TimesStops';
 import { TableType, type TimesStopsRow } from './types';
 
 type TimesStopsOutputProps = {
-  simulatedTimetableItem?: SimulationResponseSuccess;
-  simulationSummary?: SimulationSummary;
-  operationalPoints?: PathPropertiesFormatted['operationalPoints'];
+  infraId: number;
   selectedTrain?: Train;
-  path?: PathfindingResultSuccess;
+  isValid?: boolean;
```

Comment:
`isValid` is always defined (same in `useOutputTableData`)
```suggestion
  isValid: boolean;
```

Reply:
You're right, done!